code/controllers/subsystem/tcgsetup.dm

```diff
@@ -0,0 +1,10 @@
+#define CARD_FILES list("set_one.json","set_two.json")
+#define CARD_DIRECTORY "strings/tcg"
+#define CARD_PACKS list(/obj/item/cardpack/series_one, /obj/item/cardpack/resin)
```

Comment:
Turn this into a global variable instead of a define.

Reply:
gottcha. I'm keeping directory, but packs and files can move over

Reply:
it's a define mostly to prevent admemery, but it's no big deal